ruleset/rules/0365-auditd_rules.xml

```diff
@@ -1,451 +1,457 @@
 <!--
-  -  Audit rules
-  -  Created by Wazuh, Inc.
-  -  Copyright (C) 2015-2021, Wazuh Inc.
-  -  This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2.
+  Copyright (C) 2015-2021, Wazuh Inc.
 -->
 
+<!-- 
+  Audit rules for:
+    Audit rules ID:           80700 - 80792
+    Passwd rules ID:          80793 - 80799
+-->
+
+
 <group name="audit,">
 
-    <rule id="80700" level="0">
-        <decoded_as>auditd</decoded_as>
-        <description>Audit: messages grouped.</description>
-    </rule>
+  <rule id="80700" level="0">
+    <decoded_as>auditd</decoded_as>
+    <description>Audit: messages grouped.</description>
+  </rule>
 
 
-    <!-- Main audit rules -->
-    <!--
+  <!-- Main audit rules -->
+  <!--
     type=DAEMON_RESUME msg=audit(1300385209.456:8846): auditd resuming logging, sending auid=? pid=? subj=? res=success
     type=DAEMON_START msg=audit(1450875964.131:8728): auditd start, ver=2.4 format=raw kernel=3.16.0-4-amd64 auid=4294967295 pid=1437 res=failed
-    -->
-    <rule id="80701" level="1">
-        <if_sid>80700</if_sid>
-        <field name="audit.type">DAEMON_RESUME|DAEMON_START</field>
-        <field name="audit.res">success</field>
-        <description>Auditd: Start / Resume</description>
-        <group>gpg13_4.14,gpg13_10.1,gdpr_IV_30.1.g,</group>
-    </rule>
-
-    <rule id="80702" level="10">
-        <if_sid>80700</if_sid>
-        <field name="audit.type">DAEMON_RESUME|DAEMON_START</field>
-        <field name="audit.res">failed</field>
-        <description>Auditd: Start / Resume FAILED</description>
-        <group>audit_daemon,service_availability,pci_dss_10.6.1,gpg13_10.1,gpg13_4.14,gdpr_IV_35.7.d,gdpr_IV_30.1.g,hipaa_164.312.b,nist_800_53_AU.6,tsc_CC7.2,tsc_CC7.3,</group>
-    </rule>
-
-    <!--
+  -->
```

Comment:
I think that 8.d means no samples in rule file, should be placed in ini if needed.
Not sure what do we do if it's not something we added in the PR